Experimenter/Experimenter.UI/ApplicationMain.cs

```diff
@@ -54,10 +63,9 @@ static void Main(string[] args)
                                             "pluginFilePrefix must be a valid non empty string\n"+
                                             "pluginDirPath must be a valid non empty string");
             string[] pluginFiles = Directory.GetFiles(pluginDirPath, pluginFilePrefix + "*.dll");
-            if (pluginFiles.Length < 1)

```

Comment:
Here is some work to do. What happen if there is more then one *.DLL? 
I limited that it has to be one in order that i give a specific path and not seek through the project.

Reply:
I have taken account to this problem and add an optional argument --plugin-path to Experimenter.UI, which can be used to pass a specific path to plugin file. The restriction here is that the name of plugin file must have the prefix 'ExecutorPlugin'. This measure supports consistency. By default, it still searched for 'ExecutorPlugin' files in current work directory and take the first plugin file that is founded there. This is sufficient because the Experimenter-Software has trigger only one custom experiment software within an docker image.

Reply:
Not if you have multiple Experiments within one Docker Container (i.E. you are lazy and build only one).

but i'm ok with that for now.